configs/env-example

```diff
@@ -147,3 +147,8 @@ WORKERS_PORT=
 
 #Load test gateway
 LOAD_TEST_API_PORT=
+
+# CAS integration configuration
+CAS_BASE_URL=
+CAS_CLIENT_ID=
+CAS_CLIENT_SECRET=
```

Comment:
These variables need to be added to the `env-setup-deploy-secrets.yml` to allow their updates in Openshift.
Please use `GC_NOTIFY_API_KEY` as a reference to implement the changes.